src/main/tools/linux-sandbox-pid1.cc

```diff
@@ -430,6 +499,112 @@ static int WaitForChild() {
   }
 }
 
+static void MountSandboxAndGoThere() {
+  if (mount(opt.sandbox_root.c_str(), opt.sandbox_root.c_str(), nullptr, MS_BIND | MS_NOSUID, nullptr) < 0) {
+    DIE("mount");
+  }
+  if (chdir(opt.sandbox_root.c_str()) < 0) {
+    DIE("chdir(%s)", opt.sandbox_root.c_str());
+  }
+}
+
+static void CreateEmptyFile() {
+  // This is used as the base for bind mounting.
+  CreateTarget("tmp", true);
```

Comment:
Should we check the return value of CreateTarget for errors?

Reply:
Yes